lib/.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
+}
```

Comment:
you can remove this one, we already have it there
https://github.com/astahmer/openapi-zod-client/blob/3cde9645a767061cb56548a86eba1a77e1756b86/.changeset/config.json